packages/provider-elasticsearch/src/example-types/filters/step.js

```diff
@@ -0,0 +1,53 @@
+import _ from 'lodash/fp.js'
+import { pickSafeNumbers } from '../../utils/futil.js'
+
+const getMinAndMax = (range) => {
+  const min = range[0]
+  const max = range[range.length - 1]
+  return { min, max }
+}
+
+const isValueMaxOnSteps = (value, steps) => {
+  const max = steps[steps.length - 1]
+  return value === max
+}
+
+let filter = ({ field, range, steps }) => {
```

Comment:
Even though I would love to do that I'm afraid that's not possible because the object that `step.js` is returning, as well as the other files, is exactly the contract that contexture client is expecting to receive so it can work.